TenancyInformationApi.Tests/V1/E2ETests/ExampleTest.cs

```diff
@@ -1,6 +1,6 @@
-namespace BaseApi.Tests.V1.E2ETests
+namespace TenancyInformationApi.Tests.V1.E2ETests
 {
-    //For guidance on writing integration tests see the wiki page https://github.com/LBHackney-IT/lbh-base-api/wiki/Integration-Tests
+    //For guidance on writing integration tests see the wiki page https://github.com/LBHackney-IT/lbh-TenancyInformationApi/wiki/Integration-Tests
```

Comment:
This doesn't need to change

Reply:
I think this was me rather than the renaming script...i found the script didn't rename everything but perhaps I ran it wrong.

Reply:
I haven't used it before tbh, I don't have PowerShell installed.